ui/00_message/ui/clan_edit.xml

```diff
@@ -81,6 +89,9 @@
     <original>注意事項
 ・半角２文字～１２文字まで入力可能です
 ・半角英数字と一部の記号 '.,:;!?&amp;-_ が使用可能です</original>
+    <translation>Notes:
+・You can input up to 2 to 12 characters.
+・Half-size alphanumeric characters and some symbols '.,:;!?&amp;-_ can be used.</translation>
```

Comment:
Oh, this is apparently talking about non-fullwidth alphanumeric characters. I think removing the half-size part would be fine.